Assets/Scripts/PlayerInputs.cs

```diff
@@ -22,20 +22,29 @@ void Start()
 
     void Update()
     {
-        //take yes/no/neutral input (or other way of accepting exchange). 
-        if (Input.GetKeyDown(KeyCode.A))
+        //checks if the recap screen is active
+        if(manager.MissionRecapScreen.activeSelf == false)
```

Comment:
probably best to check here also for Game Over screen (dont want trades working)